### ISSUE TITLE
Wrong htaccess directory check with bedrock

### ISSUE THREAD

**Issue by mrgesco** [Sucuri/sucuri-wordpress-plugin#132]
We are using the Bedrock WordPress boilerplate installation and are facing a PHP exception.

When it tries to detect the htaccess file
```
ABSPATH = "/var/www/html/wp/"
$folder = "/var/www/html/app/uploads/sucuri"
```

```
$folder = str_replace(ABSPATH, '', $folder);
$bpath = rtrim(ABSPATH, DIRECTORY_SEPARATOR);
return $bpath . '/' . $folder . '/.htaccess';
```
Results in
```
"/var/www/html/wp//var/www/html/app/uploads/sucuri/.htaccess"
```
Instead of
```
"/var/www/html/app/uploads/sucuri/.htaccess"
```

**Comment by pierre-r** [comment]
Hello,

I have a quick fix for you.
I don't know if it will ever go live but this is my PR that should solve your problem : https://github.com/Sucuri/sucuri-wordpress-plugin/pull/156

TL;DR :
```
private static function htaccess($folder = '')
{
        if (!function_exists('get_home_path')) {
            require_once ABSPATH . 'wp-admin/includes/file.php';
        }
        $folder = str_replace(get_home_path(), '', $folder);
        $bpath = rtrim(get_home_path(), DIRECTORY_SEPARATOR);
}
```
